Lib/test/test_json/test_tool.py

```diff
@@ -11,7 +11,7 @@ class TestTool(unittest.TestCase):
     data = """
 
         [["blorpie"],[ "whoops" ] , [
-                                 ],\t"d-shtaeou",\r"d-nthiouh",
+                                 ],\t"d-shtaeou",\r"🐍 and δ",
```

Comment:
Needed something in the range U+0080-U+00FF. In Python string literals they are encoded with `\x`, but JSON always use `\u`.